src/options/index.js

```diff
@@ -1,9 +1,16 @@
+import OptSync from 'webext-options-sync';
+
+new OptSync().syncForm(document.querySelector('#options-form'));
```

Comment:
What do you think about `syncForm` accepting a CSS selector string for convenience?